scripts/env

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env bash
```

Comment:
It is not immediately obvious to me why this script would be checked in and not fetched the same way as the reporter: `curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/env` and the checksum compared. If make the fetch code and checksum verification generic we should be easily able to do that?

Reply:
They only just made the checksum for this script available and I don't think it has made its way into a release tag yet either but yeah I think it makes sense to do that.